src/tests/src/solver/optimisation/constraints/constraints_builder.cpp

```diff
@@ -51,24 +51,27 @@ struct BB
 
         addc3_netting_constraints({{.hours = {9, 10}, .globalIndex = 4, .localIndex = 0},
                                    {.hours = {11, 12}, .globalIndex = 5, .localIndex = 1}}),
-        addc1_withdrawal("addc1_withdrawal",
-                         "cluster_1",
-                         "withdrawal",
-                         "less",
-                         true,
-                         addc1_withdrawal_constraints),
-        addc2_injection("addc2_injection",
-                        "cluster_2",
-                        "injection",
-                        "greater",
-                        true,
-                        addc2_injection_constraints),
-        addc3_netting("addc3_netting",
-                      "cluster_3",
-                      "netting",
-                      "equal",
-                      true,
-                      addc3_netting_constraints)
+        addc1_withdrawal(std::make_shared<Antares::Data::ShortTermStorage::AdditionalConstraints>(
```

Comment:
Do we need **Antares::Data::ShortTermStorage** everywhere ?
It's quite cumbersome.
Indeed, here we know we're in the context of tests about short term storage.

Reply:
done